frontend/packages/console-plugin-sdk/src/codegen/plugin-resolver.ts

```diff
@@ -59,7 +67,21 @@ export const resolvePluginPackages = (
 ) => {
   const packageFiles = glob.sync(consolePkgGlob, { cwd: monorepoRootDir, absolute: true });
   const { appPackage, pluginPackages } = readPackages(packageFiles);
-  return appPackage ? pluginFilter(appPackage, pluginPackages) : [];
+
+  if (!appPackage) {
+    throw new Error(`Cannot detect Console application package ${consoleAppName}`);
```

Comment:
Note: this check was added to reflect the expectation that Console (app) itself is a plugin, i.e. `packages/console-app/src/plugin.tsx`.